composer/models/bert/model.py

```diff
@@ -87,9 +87,10 @@ def create_bert_mlm(use_pretrained: Optional[bool] = False,
         model = transformers.AutoModelForMaskedLM.from_pretrained(pretrained_model_name_or_path=pretrained_model_name,
                                                                   **model_config)
     else:
-        config = transformers.AutoConfig.from_pretrained(pretrained_model_name, **model_config)
-        assert transformers.AutoModelForMaskedLM.from_config is not None, 'AutoModelForMaskedLM has from_config method'
-        model = transformers.AutoModelForMaskedLM.from_config(config)
+        if len(model_config) == 0:
```

Comment:
```suggestion
        if not model_config:
```
this should be `None` or a `dict`. are we concerned about users passing an empty dict? (an empty dict == `None` btw)